### DISCUSSION TITLE
Possbility to always allow ssh sinage?

### DISCUSSION THREAD

**Discussion by miranyx** [quexten/goldwarden#96] [Q&A]
Hey, I wanted to ask if it is possible to deactivate the window which asks to allow ssh signage, if it is possible, I would also like to know if I can activate/deactivate it on a per app basis?

I am talking about these windows:
![20240205_09h48m38s_grim](https://github.com/quexten/goldwarden/assets/64905406/0f71eb4a-dd26-42af-b638-08f41fdad16f)
![20240205_09h48m47s_grim](https://github.com/quexten/goldwarden/assets/64905406/1a80f7ab-c841-48de-a617-d4a23e18ae03)

**Comment by quexten** [comment]
This is not possible at the moment but I have created a feature request issue tracking this:

https://github.com/quexten/goldwarden/issues/97
